packages/proxy/lib/http/response-middleware.ts

```diff
@@ -735,8 +800,8 @@ export default {
   AttachPlainTextStreamFn,
   InterceptResponse,
   PatchExpressSetHeader,
+  OmitProblematicHeaders, // Since we might modify CSP headers, this middleware needs to come BEFORE SetInjectionLevel
```

Comment:
I'd like to move the order the middleware is defined in this file to reflect the export/execution order, but it might make the PR harder to review and think I can do it in a follow up.

Reply:
Agreed, that's a lot of churn.